lib/vacuum/request.rb

```diff
@@ -36,9 +36,9 @@ class Request
 
     params 'AssociateTag' => -> { associate_tag },
            'Service'      => 'AWSECommerceService',
-           'Version'      => '2011-08-01'
+           'Version'      => -> { aws_version } || "2011-08-01"
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.